.github/workflows/build_rocks.yaml

```diff
@@ -79,7 +79,7 @@ jobs:
       - name: Build rock as tarball
         if: ${{ github.event.pull_request.head.repo.fork }}
         run: |
-          skopeo copy oci-archive:$(ls "${{ matrix.path }}"/*.rock) docker-archive:./$IMAGE_NAME.tar:$IMAGE_NAME:${{ github.run_id }}
+          skopeo copy oci-archive:$(ls "${{ matrix.path }}"/*.rock) docker-archive:./$IMAGE_NAME.tar:localhost:32000/$IMAGE_NAME:latest
```

Comment:
We are using the run_id instead of `latest` to prevent clashed when multiple runs occurs simultaneously. Note that those images might be shared using the registry.

Reply:
This is for the forked situation which uploads the OCI image as an artifact instead of using the registry. But I will just update the tag to use `run_id` to be consistent.